manifests/0000_05_config-operator_01_proxy.crd.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: proxies.config.openshift.io
+spec:
+  group: config.openshift.io
+  names:
+    kind: Proxy
+    listKind: ProxyList
+    plural: proxies
+    singular: proxy
+  scope: Cluster
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  validation:
+    openAPIV3Schema:
+      properties:
```

Comment:
@soltysh Will a description populate in `oc explain`?

Reply:
this is a known bug with the generator (top-level descriptions not being generated) that will hopefully be fixed soon when we switch to the new version of upstream controller-tools